### ISSUE TITLE
Remove 1 Vote Maps from Weighted Votes

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#7897]
(WEB REPORT BY: gunosaguki REMOTE: 158.69.55.2:7777)
Its not worth potentially allientating a large portion of the active players who clearly didnt want the 1 vote map for that round.

I'm fine for having maps having a chance to be vote. but 1 vote is NOT a "chance" its a high chance of it being a special snowflake who was just looking to piss people off by it winning.

**Comment by 9600bauds** [comment]
>1 vote is NOT a "chance"
It's a small chance, that sounds like a chance to me

>a special snowflake
hooo boy

**Comment by Ephx1** [comment]
Haaaaaaaaaaaaaaaaaaaaaahahaha

**Comment by Zth--** [comment]
# **what**

**Comment by Exxion** [comment]
[Honk]

**Comment by Imagebard** [comment]
Wew lad
:-1:

**Comment by CptWad** [comment]
I admit its bullshit but it still wont get changed. I don't think it should. a 5% chance of winning is bullshit but if it forces you to play maps you don't like then you can post the problems you have with said map and hope they get fixed.
Unless its deff
Fuck that SME room

**Comment by Ephx1** [comment]
> Spelling/Grammar

It just gets better and better

**Comment by SysadminJeroen** [comment]
# [GET FUCKED](https://raw.githubusercontent.com/d3athrow/vgstation13/Bleeding-Edge/sound/items/bikehorn.ogg)

**Comment by Gunosaguki** [comment]
CptWad, other maps get played plenty now. im just talking about 1 votes for that PARTICULAR vote. There is not much point in a 1 vote map winning if clearly hardly anyone wanted to play that map at that moment.

There is value to be had to balancing the "We need to test these maps!" and "We want to play THIS map".

**Comment by Exxion** [comment]
Which is why it almost never happens
The chance of a map winning is exactly equal to the percentage of people who voted for it, as it should be
And to be honest I can't figure out any reason one would want it to be different other than just so we can go back to 24/7 Box, which we implemented this feature to avoid.

**Comment by CptWad** [comment]
Ye. Didn't we have a [Wontfix] tag?
Its working as intended, if you start noticing 1 votes winning more often then they should then come back and see if the randomness is not actually random

**Comment by Duny-** [comment]
This has such a tiny chance of happening I don't understand why you're even complaining

**Comment by Gunosaguki** [comment]
It wouldnt make it 24/7 box. it would give % to the non-box maps too.

Im honestly shocked at how much against this everyone is. I feel like most of you are thinking about this from a map testing point of view and not a player happiness point of view.

**Comment by Exxion** [comment]
You're one of about three people who dislikes the new system, as far as I'm aware.

**Comment by 9600bauds** [comment]
Apparently it's allientating to have your vote not win, so his complaint is that one vote winning is allientating a lot of people.
However he's not taking into consideration that said voter probably got very allientated when his vote didn't win the last 30 times, the current system keeps the allientation equally distributed. Otherwise you would be permanently allientating the lone voter.

**Comment by Exxion** [comment]
If you really, really cannot tolerate the map that wins, then you can just sit out a round. But honestly the maps that get played less are better in my opinion and even if you disagree it seems rather petty to get that upset over just which map is played.

**Comment by Gunosaguki** [comment]
i dont dislike any of the maps except for non-snow taxi, but ill tollerate it because it has the best roid and i play xenoarch/miner often.. I'm purely talking about this from a "This looks like a stupid ass part of this system, we should probably tweak it a bit" perspective. i wont get really pissed off it isn't changed. This is mostly from a "I think it could be improved" point of view.

**Comment by Exxion** [comment]
But what's wrong with it? I just don't get it.

**Comment by 9600bauds** [comment]
I don't think you appreciate the allientation distribution

**Comment by Gunosaguki** [comment]
Whats wrong with it is that there should be some factoring of the larger playerbase in the weighted vote.

**Comment by 9600bauds** [comment]
The votes control the % of each map winning

**Comment by Gunosaguki** [comment]
You're missing the point entirely. removing 1 votes from the weighted vote would ENSURE that voting has some purpose other than "rng dictated this map won" it makes it so that the maps that were picked were wanted to be playn by more than one person.

**Comment by Exxion** [comment]
RNG dictated the map won with a percent chance equivalent to the percentage of people who want to play it
Exactly as it should be

**Comment by Gunosaguki** [comment]
why are we allowing RNG to pick a map with 1 vote? thats the point im making. for the sake of RNG? thats it?

**Comment by Exxion** [comment]
Because one person voted for it, of course
The point you're making just doesn't seem to make any sense

**Comment by PJB3005** [comment]
Becuase if 10% of the playerbase wants to play deff then deff gets picked 10% of the time?

**Comment by Gunosaguki** [comment]
the point you're making doesnt seem to make any sense

**Comment by 9600bauds** [comment]
I'm lost here, do you have an issue with RNG or with minority votes?

**Comment by Gunosaguki** [comment]
1 person =/= minority is the point im making

1 person = individual

**Comment by Exxion** [comment]
Which is still a minority

**Comment by Gunosaguki** [comment]
No. there is majorities. there is minorities. there are individuals that nobody agrees with

**Comment by 9600bauds** [comment]
You seem to be the only individual here who doesn't like the new system

**Comment by Exxion** [comment]
So you're suggesting that someone's opinion is invalid just because nobody who decided to vote agrees with them?

**Comment by Gunosaguki** [comment]
You guys seem to not understand what im getting at all. and its not even remotely hard to understand but you seem to have trouble with it

**Comment by Duny-** [comment]
You're suggesting to make the vote unfair because something that happens once every year makes you mad for a round.

**Comment by Gunosaguki** [comment]
1 vote is unfair. I dont give a fuck which map won the 1 vote

**Comment by Exxion** [comment]
How is it unfair to give 1 vote a fair chance of winning?
How is it fair to discount that one vote?

**Comment by Duny-** [comment]
It is perfectly fair. You should go back to school and learn about statistics.

**Comment by Gunosaguki** [comment]
1 vote is more unfair winning to the other maps with 3+ votes  than it is to the 1 vote only having 1 vote and thus a small chance

**Comment by Exxion** [comment]
That's not how fairness works.

**Comment by Gunosaguki** [comment]
you guys need to get the statistics out of your eyes

**Comment by Gunosaguki** [comment]
Unlikely =/= its good to leave it as is

Cancer is unlikely. is that good?

**Comment by PJB3005** [comment]
You do realise that the chance for a map to win is directly related to the amount of votes, right?

Also, cancer is like the shittiest analogy ever.

**Comment by Gunosaguki** [comment]
i just picked a random bad thing

**Comment by Exxion** [comment]
You know, if we did decide to discount single votes because nobody agrees with them, then your opinion would be the first to be discounted.

**Comment by Duny-** [comment]
You need to literally learn about statistics. The other 95 times it's the other maps which got picked.

**Comment by Gunosaguki** [comment]
except at least 3 people int he thread have agreed with me so im not a single vote

**Comment by Duny-** [comment]
So you agreed with yourself anonymously twice, that doesn't make your argument any smarter.

**Comment by Gunosaguki** [comment]
Lol. you immediately just made me think way less of you by claiming people on my side are samefags

**Comment by Gunosaguki** [comment]
that explains why you cant see a point

**Comment by Exxion** [comment]
1/20 = 3/60
60 is, of course, an estimate, but probably a low one given how many people have visibly spoken on this issue.

**Comment by 9600bauds** [comment]
5% to win = 1 in 19 chance to win -> map wins -> 19 people played a map they didn't want to

5% to win = 1 in 19 chance to win -> map doesn't win 19 times in a row -> guy played a map he didn't want to 19 times

It evens out

**Comment by Duny-** [comment]
I couldn't care less what someone who wants snowflake code to be added just so he doesn't get mad once a year thinks about me.

**Comment by Gunosaguki** [comment]
9600bauds made the best point

**Comment by clusterfack** [comment]
You know by your own system, your vote would have zero chance of getting this system changed

**Comment by Gunosaguki** [comment]
clusterfack. that'd only be on github

**Comment by Exxion** [comment]
9600bauds made the same point the rest of us have been
He just explained it more clearly.

**Comment by 9600bauds** [comment]
We should go back to the old system which spoke like a vox

**Comment by PJB3005** [comment]
Also 3 people in the thread is the minority in this case.

**Comment by Gunosaguki** [comment]
PJB3005, but the point im making is individual votes. not minority votes

**Comment by ghost** [comment]
@9600bauds 30% chance of winnings 
I could trust the vox to fudge the numbers in favour of defficiency.  VOX VOTES WHEN.

**Comment by PJB3005** [comment]
![](http://i.imgur.com/dnOtdNX.png)

**Comment by Exxion** [comment]
Also, an individual vote on highpop is a TOTALLY different animal than an individual vote on deadpop

**Comment by 9600bauds** [comment]
So a 5% vote is fine if it's 3 people in 60, but a 5% vote is bad if it's 1 person in 20?

**Comment by ghost** [comment]
Lads hold on what happens if there's 4 people on the server and they all vote different maps

**Comment by Kurfursten** [comment]
Alternate proposal: votes are discarded after each vote, BerlinStation is selected automatically each round.

**Comment by ghost** [comment]
Do they go to purgatory station?

**Comment by Gunosaguki** [comment]
DrCelt you havin a chuckle?

**Comment by ghost** [comment]
Yes

**Comment by 9600bauds** [comment]
I chuckled too

**Comment by PJB3005** [comment]
I was thinking the same thing as celt but decided not to say it because I'd sound like an asshole.

**Comment by Gunosaguki** [comment]
these latter points are better arguments vs. my proposal

**Comment by Gunosaguki** [comment]
high pop 1 vote is way worse
1 vote during 10-20 votes is admitedly not as bad

**Comment by clusterfack** [comment]
Sounds like that one guy who thinks XCOM is rigged when his 95% shot misses

**Comment by ghost** [comment]
@kurfursten Can you rename bus station to Limo Station? I far prefer the name as limo is like a fancy taxi whereas a bus is a different thing than a taxi

**Comment by 9600bauds** [comment]
Wasn't the episode that instigated this issue in the first place a 5% vote?

**Comment by Ephx1** [comment]
Alright cluster you've gone too far, XCOM IS rigged, have you seen those fucking thin men?

**Comment by Exxion** [comment]
> Tab out for two minutes
> Seven notifications

I have made a terrible mistake

**Comment by Kurfursten** [comment]
@DrCelt Yeah the name isn't set in stone, it could be Uber station for all I care

**Comment by PJB3005** [comment]
![](http://i.imgur.com/fbOqHNs.png)

**Comment by Duny-** [comment]
Don't start talking about thin men, fucking put triggers in the tags

**Comment by Gunosaguki** [comment]
Ultimately I think while its pretty bad, it would be pretty pointless to change it for the rare circumstance. its moreso an issue of "why bother fixing this?" than it is "yes it would be better probably"

**Comment by 9600bauds** [comment]
I can't even tell what we're arguing about anymore, I'm gonna go do human being things

**Comment by ghost** [comment]
@9600bauds is a changeling please lynch he

**Comment by Gunosaguki** [comment]
End of story is i dont like 1 votes winning. thats my arugment. map that got 1 vote doesnt matter. tickles my autism

**Comment by Kurfursten** [comment]
blood test when?

**Comment by 9600bauds** [comment]
humanposters go and stay go

**Comment by SysadminJeroen** [comment]
Let's just declare @Gunosaguki's vote invalid because he is the only one who voted :+1: on this issue, it seems that everyone either abstains or :-1:.

**Comment by ghost** [comment]
;g I convinced the coderbase @9600bauds is a ling

**Comment by 9600bauds** [comment]
;g absorbing 3 in dorms

**Comment by Gunosaguki** [comment]
I forgot to turn off email notifications. oh god

**Comment by PJB3005** [comment]
You're admitting that this is 100% subjective, the statistics are objective.

**Comment by 9600bauds** [comment]
Let's just have a nice round of Beepsky Smash in dorms and forget this happened

**Comment by Exxion** [comment]
:g Absorbing memes on GitHub

**Comment by Imagebard** [comment]
![1428767259915](https://cloud.githubusercontent.com/assets/7571238/12626752/f832680c-c518-11e5-8209-4a897e86b177.jpg)

94 posts within 2 hours, and nobody agrees with you except yourself.
Are you done getting us to laugh at you yet?

**Comment by Exxion** [comment]
Yes, I think he's done considering he hasn't replied in 15 minutes
Leave him be

**Comment by 9600bauds** [comment]
Remove smug anime from the premises

**Comment by ghost** [comment]
Did someone say smug anime?
![are you frustrated](https://cloud.githubusercontent.com/assets/7193289/12626974/4d1edd22-c533-11e5-8c41-559d7fc3c42d.jpg)

**Comment by 9600bauds** [comment]
the era of reaction images in github dawns over us

**Comment by Exxion** [comment]
Is that a fucking space before that question mark?

**Comment by 9600bauds** [comment]
smug anime man confirmed railposter

we are nearing meme singularity

**Comment by Duny-** [comment]
Looks like it is. Damn. I'm french and I'm literally supposed to put spaces before my question marks, and even I don't do this.

**Comment by ghost** [comment]
Does that frustrate you, @Exxion ? 

I'm a dumby

**Comment by Kurfursten** [comment]
![memes](https://cloud.githubusercontent.com/assets/9782036/12627085/831a40a6-c501-11e5-9017-33bf073b38f7.png)

**Comment by Exxion** [comment]
It frustrates me that a translator did that, yes

**Comment by ghost** [comment]
Who repeatedly posts

**Comment by ghost** [comment]
Whoops, accidentally uploaded the same one, sorry. 
The same image over and over

**Comment by Exxion** [comment]
You have to meme harder, Celt

**Comment by NigglyWiggly** [comment]
> Whats wrong with it is that there should be some factoring of the larger playerbase in the weighted vote.

Jesus fucking christ do you read what you write

**Comment by Exxion** [comment]
Are you fucking posting every frame?

**Comment by ghost** [comment]
Constantly

**Comment by NigglyWiggly** [comment]
Celt you could at least not shitpost. You're masking the actual stupidity here with your own.

**Comment by 9600bauds** [comment]
![unaware](https://cloud.githubusercontent.com/assets/6526157/12627186/400946a8-c51b-11e5-818c-45cecef67be8.jpg)

it is an analogy for the state of this thread

**Comment by ghost** [comment]
3

**Comment by AlphaBuffalo** [comment]
I had this opened before I went to the store
come back
it explodes

**Comment by gbasood** [comment]
did you know
1 in every 100 spessmen doesn't like fun

**Comment by Exxion** [comment]
In my experience it's more like 60 in every hundred

**Comment by ghost** [comment]
WAS MY COMMENT EDITED? TYRANNY.

**Comment by 9600bauds** [comment]
wasn't me

**Comment by ghost** [comment]
![what did you say you little bitch](https://cloud.githubusercontent.com/assets/7193289/12627904/62908142-c538-11e5-9ea7-fda4aa8e13ac.jpg)

**Comment by ghost** [comment]
Who thought this warranted 120 responses.

Here's one more.

**Comment by 9600bauds** [comment]
We were having fun memeing it up and beating a dead horse, should do it again sometime

**Comment by ihadtoregisterforthis** [comment]
Cancer is very probable too

**Comment by Imagebard** [comment]
He already said he'd _only_ be outnumbered in the github
You know very well he's going to try again on the main serve

**Comment by Gunosaguki** [comment]
Nah i won't. At most I may make a snide OOC comment whenever 1 votes win, but i probably wont bother trying anymore.

**Comment by Zth--** [comment]
I'll forget about this thread if OP tells me a truly evil adminbus plan

**Comment by Gunosaguki** [comment]
Everyone gets a Pet, but centcom will only allow one to be taken on the shuttle.

Thats the first thing to come to mind because I actually have never thought of any bus ideas before.

**Comment by SonixApache** [comment]
Is this really ~100 comments about people not understanding how chance calculation works?

**Comment by PJB3005** [comment]
Yes

**Comment by IratePirate** [comment]
This is awful.

**Comment by PJB3005** [comment]
What this thread or the idea?

**Comment by Gunosaguki** [comment]
He's probably talking about both. after sitting back and playing more and seeing more rounds where only 4-5 people voted PERIOD. this idea definately wouldnt work. Being right or wrong has nothing to do with it when we dont have enough people voting.

**Comment by fitrandiddle** [comment]
You should probably leave it. The coders are too attached to their new maps to ever stop pushing them.

**Comment by Exxion** [comment]
Did you sign up just to complain about non-Box maps existing?

**Comment by 9600bauds** [comment]
except that in this case it's the playerbase voting for those maps

**Comment by fitrandiddle** [comment]
I signed up specifically because coders were demanding in thread people sign up and tell them problems with the various maps.
Still waiting on my public medical bay scanners on all maps.

**Comment by 9600bauds** [comment]
@Duny- he's right desu

**Comment by d3athrow** [comment]
fun fact the function used in the weighted maps doesn't actually do weighted picking it just picks something at random

**Comment by Ephx1** [comment]
RIGGED

**Comment by Exxion** [comment]
Why is this issue still open

**Comment by clusterfack** [comment]
Because I forgot about it

**Comment by Probe1** [comment]
@fitrandiddle there aren't public advanced scanners because they're advanced scanners.  that's medbay for you.  If you make an issue and people unanimously say "Yeah add this!" I would but no ones asking for it in a place I can see and in general it's asking for medbay stuff to not be in medbay which is weird and contradictory design.

Yes I realize this is a 2 month old comment.
